src/plugins/orangehrmRecruitmentPlugin/Api/VacancyAttachmentAPI.php

```diff
@@ -258,6 +327,40 @@ public function getValidationRuleForGetOne(): ParamRuleCollection
     }
 
     /**
+     * @OA\Put(
+     *     path="/api/v2/recruitment/vacancies/{vacancyId}/attachments/{attachmentId}",
+     *     tags={"Recruitment/Vacancy Attachments"},
+     *     @OA\PathParameter(
+     *         name="attachmentId",
+     *         @OA\Schema(type="integer")
+     *     ),
+     *     @OA\PathParameter(
+     *         name="vacancyId",
+     *         @OA\Schema(type="integer")
+     *     ),
+     *     @OA\RequestBody(
+     *         @OA\JsonContent(
+     *             @OA\Property(property="currentAttachment", type="string"),
+     *             @OA\Property(property="attachment", ref="#/components/schemas/Base64Attachment"),
+     *             @OA\Property(property="vacancyId", type="integer"),
+     *             @OA\Property(property="comment", type="string"),
+     *             @OA\Property(property="attachmentType", type="integer")
```

Comment:
can add `maxLength` and `maximum`
```suggestion
     *             @OA\Property(
     *                 property="comment",
     *                 type="string",
     *                 maxLength=OrangeHRM\Recruitment\Api\VacancyAttachmentAPI::PARAM_RULE_COMMENT_MAX_LENGTH
     *             ),
     *             @OA\Property(
     *                 property="attachmentType",
     *                 type="integer",
     *                 maximum=OrangeHRM\Recruitment\Api\VacancyAttachmentAPI::PARAM_RULE_ATTACHMENT_TYPE_MAX_LENGTH
     *             )
```